ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/util/DiscoverRemoteHostsUtil.java

```diff
@@ -42,6 +42,7 @@ private DiscoverRemoteHostsUtil() {
         .map(QueryMetadata::getAllMetadata)
         .filter(Objects::nonNull)
         .flatMap(Collection::stream)
+        .filter(streamsMetadata -> !streamsMetadata.hostInfo().equals(HostInfo.unavailable()))
```

Comment:
This seems to capture the filtering check from here: https://github.com/confluentinc/ksql/commit/9615a0296bfacd6eea9f054958319987792865f4#diff-b5d09835b101449b87f099018515be248e21a66ffb0affae62041070ccd7ff55L47